pkgs/development/python-modules/pyroma/default.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, docutils, pygments
+}:
+
+buildPythonPackage rec {
+  pname = "pyroma";
+  version = "2.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "00j1j81kiipi5yppmk385cbfccf2ih0xyapl7pw6nqhrf8vh1764";
+  };
+
+  buildInputs = [ docutils pygments ];
```

python packages should be listed as propagatedBuildInputs,

according to https://github.com/regebro/pyroma/blob/d74a08b8dac558103ff14d760629151b9af56b5b/setup.py#L38
```suggestion
  propagatedBuildInputs = [ docutils pygments setuptools ];
```

---

otherwise PR LGTM

---

Didn't realize I needed to include `setuptools` as well.  Thanks!